### PR TITLE
Update .editorconfig

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-[*.py]
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
 indent_style = space
+insert_final_newline = true
 trim_trailing_whitespace = true
-indent_size = 4
-end_of_line = LF
```
Comment (StevenBlack) [inline .editorconfig:7]: @XhmikosR as far as you know, will this change on L7 affect Windows users?

Comment (StevenBlack) [reply .editorconfig:7]: @XhmikosR Everything else LGTM.

Comment (XhmikosR) [reply .editorconfig:7]: It will still be LF inherited from the first block like it was before

Comment (StevenBlack) [reply .editorconfig:7]: @XhmikosR Duh! Of course. Thanks.

**Comment by StevenBlack** [comment]
Thank you @XhmikosR!

Merging!

**Pull request merged**